### PR TITLE
(MODULES-2244) Remove WaitFor DSC Resources from Upstream

### PR DESCRIPTION
**Pull request by jpogran**
Update rake task to remove any WaitFor DSC Resources downloaded from the
upstream DSCResources repo

### CODE REVIEW

**Comment by ferventcoder** [comment]
:shipit:

**Comment by ferventcoder** [comment]
Waiting for appveyor/travis

**Comment by ferventcoder** [comment]
I don't think they will be affected, but just in case.

File: build/dsc.rake [hunk 1]
```diff
@@ -110,8 +110,8 @@ eod
     task :build, [:module_path] do |t, args|
       module_path = args[:module_path] || default_dsc_module_path
       m = Dsc::Manager.new
-      wait_for_resources = Dir["#{module_path}/**/*WaitFor*"]
-      fail "WaitFor* resources found - aborting type building!\n\n#{wait_for_resources}\n" if !wait_for_resources.empty?
+      wait_for_resources = Dir["#{module_path}/**/MSFT_WaitFor*"]
+      fail "MSFT_WaitFor* resources found - aborting type building! Please remove any MSFT_WaitFor* DSC Resources and run the build again.\n\n#{wait_for_resources}\n" if !wait_for_resources.empty?
```
Comment (Iristyle) [inline build/dsc.rake]: Since we're dumping the list of resources, can probably reword:

`Please remove any`

To

`Please remove the following`

Comment (ferventcoder) [inline build/dsc.rake]: Ah true.

**Comment by Iristyle** [comment]
:+1:

**Pull request merged**